hack/k8s/example-coord.template.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: datamon-coord-demo
+spec:
+  selector:
+    matchLabels:
+      app: datamon-coord-demo
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: datamon-coord-demo
+    spec:
+      initContainers:
+      - name: init-application-wrap
+        image: gcr.io/onec-co/datamon-fuse-demo-coord-datamon:latest
+        imagePullPolicy: "Always"
+        command: ["sh", "-c", "mkdir /tmp/coord/.scripts; cp wrap_application.sh /tmp/coord/.scripts/wrap_application.sh; chmod a+x /tmp/coord/.scripts/wrap_application.sh; echo 'placed application wrapper'"]
+        stdin: true
+        tty: true
+        volumeMounts:
+        - mountPath: /tmp/coord
+          name: container-coord
+      containers:
+      - name: demo-app
+        image: gcr.io/onec-co/datamon-fuse-demo-coord-app:latest
+        imagePullPolicy: "Always"
+        # todo: determine proper parameterization
```

Comment:
@galvare2 , these todos about the parameters to the `wrap_*.sh` a remaining uncertainty.

there's also the naming convention on the `volumeMounts`:  place them in `/tmp/`? `/`? `/datamon/`?  make the naming convention configurable via parameters to the script?

there are probably some other things to review here, yet despite the [wip] tag, this is ready for review when you have time.